.buildkite/pipeline.yml

```diff
@@ -44,21 +44,21 @@ steps:
    - nix-build tests/tezos-binaries.nix --no-out-link --arg path-to-binaries ./docker
    branches: "!master"
 
- - label: test deb source packages via docker
-   commands:
-   - eval "$SET_VERSION"
-   - nix run -f. pkgs.docker -c ./docker/docker-ubuntu-packages.sh source
-   artifact_paths:
-     - ./out/*
-   branches: "!master"
- - label: test deb binary packages via docker
-   commands:
-   - eval "$SET_VERSION"
-   # Building all binary packages will take significant amount of time, so we build only one
-   # in order to ensure package generation sanity
-   - nix run -f. pkgs.docker -c ./docker/docker-ubuntu-packages.sh binary tezos-baker-006-PsCARTHA
-   - rm -rf out
-   branches: "!master"
+ # - label: test deb source packages via docker
+ #   commands:
+ #   - eval "$SET_VERSION"
+ #   - nix run -f. pkgs.docker -c ./docker/docker-ubuntu-packages.sh source
+ #   artifact_paths:
+ #     - ./out/*
+ #   branches: "!master"
+ # - label: test deb binary packages via docker
+ #   commands:
+ #   - eval "$SET_VERSION"
+ #   # Building all binary packages will take significant amount of time, so we build only one
+ #   # in order to ensure package generation sanity
+ #   - nix run -f. pkgs.docker -c ./docker/docker-ubuntu-packages.sh binary tezos-baker-006-PsCARTHA
+ #   - rm -rf out
+ #   branches: "!master"
```

Comment:
Is this relevant?

Reply:
I'll drop commit with these changes, they're here just to speed up the CI, since these steps are quite slow